tests/scripts/run-xunit-perf.cmd

```diff
@@ -2,13 +2,14 @@
 @rem The .NET Foundation licenses this file to you under the MIT license.
 @rem See the LICENSE file in the project root for more information.
 
-@echo off
-@if defined _echo echo on
+@if not defined _echo echo off
 
 setlocal ENABLEDELAYEDEXPANSION
   set ERRORLEVEL=
   set BENCHVIEW_RUN_TYPE=local
   set CORECLR_REPO=%CD%
+  set LV_SANDBOX_DIR=%CORECLR_REPO%\sandbox
```

Comment:
Does it make more sense to have the defai;t sandbox live under `bin` with similar derived / built stuff?